common/djangoapps/course_modes/models.py

```diff
@@ -548,7 +567,7 @@ def min_course_price_for_currency(cls, course_id, currency):
         return min(mode.min_price for mode in modes if mode.currency.lower() == currency.lower())
 
     @classmethod
-    def enrollment_mode_display(cls, mode, verification_status):
+    def enrollment_mode_display(cls, mode, verification_status, course_id):
```

Comment:
Can we move all of this logic to the view? Why does the model care about display values?